better_html.gemspec

```diff
@@ -21,17 +21,17 @@ Gem::Specification.new do |s|
     "allowed_push_host" => "https://rubygems.org"
   }
 
+  s.extensions    = ['ext/better_html_ext/extconf.rb']
   s.files = Dir["{app,config,db,lib,ext}/**/*", "MIT-LICENSE", "Rakefile", "README.rdoc"]
   s.test_files = Dir["test/**/*"]
-  s.require_paths = ["lib"]
+  s.require_paths = ["lib", "ext"]
```

Comment:
Do I need this? I think the `.so` file moves to `lib`.

Reply:
You don't need to do this, see e.g. https://github.com/bcrypt-ruby/bcrypt-ruby/blob/master/bcrypt.gemspec#L13

Reply:
Pushed a commit removing it